sdcm/cluster.py

```diff
@@ -4677,6 +4678,7 @@ def kill_docker_loaders(self):
         for loader in self.nodes:
             try:
                 loader.remoter.run(cmd='docker ps -a -q | docker rm -f', verbose=False, ignore_status=True)
+                self.log.info("Killed docker loader on node: %s", loader.name)
```

Comment:
please don't use info, we have enough print on this level, debug is good enough for this.

Reply:
killing loaders is a very rare thing and important part of a test flow. I'd rather keep it at info level.

I agree, log info level is polluted - we should have some task to clean it up from unimportant things.